README.md

```diff
@@ -172,6 +172,18 @@ This event was mislabeled \#46 on Meetup.
 
 2012-12-11 | 18:30 | [Meetup Event](https://www.meetup.com/Berlin-Hack-and-Tell/events/94195282/) | [Photos](https://www.meetup.com/Berlin-Hack-and-Tell/photos/12140712/) | [@c-base](https://www.c-base.org)
 
+* Bomberman multiplayer 3d by Stefan
+* [sq.rotogame.com](https://github.com/feklee/rotogamesq) (rotating puzzle game in JS/node/socket io) by [Felix](http://twitter.com/feklee)
+* [NoFlo](https://github.com/noflo/noflo) (JavaScript implementation of Flow-Based Programming) by [Henri](https://github.com/bergie)
+* radvisor.net (Resident Advisor for mobile) by Antonio
+* Location Sharing (via websockets) by Tim
+* [Arduino/GPRS battery voltage monitor](https://github.com/uwekamper/BatteryMonitor/) by [Uwe](http://uwekamper.de)
+* ["Chirp"](https://wemakethings.net/chirp/) Plant Watering Alarm by [Albert](https://twitter.com/miceuz)
+* codebrowser - web-based code browser by Oliver
+* "epos" embedded persistent object store (think NoSQL) in Go by Andreas
+* ["Trelloviz"](http://nitram509.github.io/trelloviz/) cumulative flow diagrams for Trello by [Martin](https://twitter.com/nitram509) ( [slides](http://nitram509.github.io/preZENtations/trelloviz-introduction/index.html)
```

Comment:
Maybe we should agree on a common line format?

For the entries till now, we linked the project and the description (larger link surface), and separated them via "-", and didn't use quotes. We used parentheses for additional links (e.g. to GitHub, slides), and but them before "by".

Line 184 then would look like this:

```
* [Trelloviz - cumulative flow diagrams for Trello](http://nitram509.github.io/trelloviz/) ([Slides](http://nitram509.github.io/preZENtations/trelloviz-introduction/index.html)) by [Martin](https://twitter.com/nitram509)
```

(Note that I don't want to put that style in stone, but if we decide to use a different style, we should change this for all entries. Also, I was thinking about putting this information into a JSON file, and using a Jekyll template to generate the BHNT page from that. This should all work out of the box on GitHub pages. Having a common style would facilitate creating that JSON.)